packages/reactivity/src/dep.ts

```diff
@@ -1,6 +1,10 @@
-import { ReactiveEffect, trackOpBit } from './effect'
+import { ReactiveEffect, removeEffectFromDep, trackOpBit } from './effect'
 
-export type Dep = Set<ReactiveEffect> & TrackedMarkers
+export type Dep = Set<ReactiveEffect> &
+  TrackedMarkers & {
+    target?: unknown
+    key?: unknown
+  }
```

Comment:
I wasn't sure if there was a reason why the properties `w` and `n` use single-letter names. I've used `target` and `key` here, but they can easily be changed to `t` and `k` if required.